backend/compact-connect/lambdas/common-python/cc_common/data_model/client.py

```diff
@@ -222,13 +223,16 @@ def _generate_privilege_record(
         jurisdiction_postal_abbreviation: str,
         license_expiration_date: date,
         compact_transaction_id: str,
+        original_issuance_date: date | None = None,
     ):
+        today = datetime.now(tz=UTC).date()
         privilege_object = {
             'providerId': provider_id,
             'compact': compact_name,
             'jurisdiction': jurisdiction_postal_abbreviation.lower(),
             'status': 'active',
-            'dateOfIssuance': datetime.now(tz=UTC).date(),
+            'dateOfIssuance': original_issuance_date if original_issuance_date else today,
+            'dateOfRenewal': today,
```

Comment:
UI: What's going on here? (I'm on the mock API, so that might be it.) Confused by this page.

<img width="1511" alt="image" src="https://github.com/user-attachments/assets/d6261fe4-f7ac-4787-8675-23b8083d11f6">

@jsandoval81

Reply:
This is a stand in for implementing the second half of the flow before we had the language / designs nailed down. We will be going back and adding this functionality in.

Reply:
Got it, thanks!